utils/dynamicURLs.js

```diff
@@ -0,0 +1,8 @@
+export const DYNAMIC_URL = {  //different endpoints for fetching different categories
+    hot:{
+      boardgame: "/hot?type=boardgame",
+      rpg : "/hot?type=rpg",
+      videogame : "/hot?type=videogame"
+    }
+  }
```

Comment:
I really like it that you put this into a utils folder. Maybe you wanna do the same with the "fetcher" function and the "parseXML" function from your endpoint?

Reply:
we are going to change it later